app/services/audio_processing_service.py

```
@@ -177,98 +194,130 @@ def process_audio_task(
+                        "result": result,
+                        "duration": duration,
+                        "start_time": start_time,
+                        "end_time": end_time,
+                        # Clear any error recorded by a previous failed attempt.
+                        "error": None,
+                    },
+                )
+
+                return
+
+            except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)